webapp/src/js/services/enketo.js

```diff
@@ -205,6 +205,10 @@ angular.module('inboxServices').service('Enketo',
       return LineageModelGenerator.contact(contact._id)
         .then(function(model) {
           return model.lineage;
+        })
+        .catch(function(err) {
+          $log.warn(`Failed to get lineage of contact: ${contact._id}`, err);
+          return [];
```

Comment:
I think this is too permissive - we should only do this if the contact was not found. If it fails for any other reason the `err` should be thrown. Can you check for a 404 code or something?

Reply:
Yep!